src/main/java/Event.java

```diff
@@ -7,6 +7,16 @@
     protected LocalDate by;
```

Comment:
I like that you made this protected so that nobody can change the date for the Event (and I see you did the same for the Deadline class also).